merge-release-run.js

```diff
@@ -57,8 +57,13 @@ const run = async () => {
   console.log('current:', current, '/', 'version:', version)
   let newVersion = execSync(`npm version --git-tag-version=false ${version}`).toString()
   console.log('new version:', newVersion)
-  exec(`npm publish --access=public`)
-  exec(`git checkout package.json`) // cleanup
+
+  let access = ''
```

Comment:
This can be more easily written as `const access = process.env.NPM_PRIVATE ? ‘restricted’ : ‘public’`